client/packages/android/app/src/main/java/org/openmsupply/client/ErrorPage.java

```diff
@@ -0,0 +1,6 @@
+package org.openmsupply.client;
+
+public class ErrorPage {
+    // the unhappy man svg and the text "Oops! Something's gone wrong"
```

Comment:
Would be great to have some instructions of how this was made, and what about splash screen can splash screen be encoded this way ?